intern_program/intern_program_fall_2018_projects.md

```diff
@@ -34,3 +34,30 @@ This includes building a `.json` catalog of the existing datasets, integration a
 
 #### Skills Required
 Python programming, basic knowledge of html
+
+
+### Building and analyzing 3D virtual terrain models of lava domes from lab experiments and the field
+
+#### Supervisor
+Einat Lev, Lamont Associate Research Professor
+
+#### Description
+Lava domes form when highly viscous lava erupts from a volcano and piles up above the source vent. Domes are higly unstable and collapse frequently, forming dangerous currents of ash and debris. This summer we performed a series of experiments where we simulated lava dome emplacement using wax+clay slurries, and studied the influence of different parameters on the structure of the resulting dome. We collected lots of pictures of the wax domes, and now would like to use photogrammetry to create virtual 3D models of them. Then, the student will apply the same skills to constructing 3D terrain models of lava domes and flows we colelcted using drones in Indonesia, Ecuador, and Hawai'i. All the models will be analyzed for surface fractures and roughness using Matlab/Python image processing tools and QGIS/ArcGIS mapping tools. 
```

Comment:
Can we clarify whether the intern is being asked to develop new codes / data analysis workflows? Or would they be applying an existing workflow to new datasets? This would help determine the level of expertise required.

Reply:
The intern will work with existing codes and program and will apply existing workflows.